docs/gettingstarted/modfiles.md

```diff
@@ -248,3 +248,39 @@ An entry in `neoforge.mods.toml` does not need a corresponding `@Mod` annotation
 [versioning]: versioning.md
 [enumextension]: ../advanced/extensibleenums.md
 [featureflags]: ../advanced/featureflags.md
+## Mod files tree
+```
+root
+ |-- .gradle
+ |-- gradle
+ |-- build
+ |-- run
+ |-- src
+ |    |-- main
+ |    |    |-- java
+ |    |    |    |-- <com/net>
+ |    |    |    |    |-- <username>
+ |    |    |    |    |    |-- <modid>
+ |    |    |    |    |    |    |-- <some packages and single files>
+ |    |    |    |    |    |    |-- <modname>.java
+ |    |    |    |    |    |    |-- config.java
```

Comment:
Again, can't assume this file structure, or are you trying to show the example project, in which case this still does not match.